java-frontend/src/main/java/org/sonar/java/se/MethodYield.java

```diff
@@ -19,29 +19,87 @@
  */
 package org.sonar.java.se;
 
+import org.sonar.java.se.constraint.BooleanConstraint;
 import org.sonar.java.se.constraint.Constraint;
+import org.sonar.java.se.constraint.ObjectConstraint;
+import org.sonar.java.se.symbolicvalues.SymbolicValue;
 
 import javax.annotation.Nullable;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class MethodYield {
   Constraint[] parametersConstraints;
   int resultIndex;
   @Nullable
   Constraint resultConstraint;
+  boolean exception;
 
   public MethodYield(int arity) {
     this.parametersConstraints = new Constraint[arity];
     this.resultIndex = -1;
     this.resultConstraint = null;
+    this.exception = false;
   }
 
   @Override
   public String toString() {
     return "{params: " + Arrays.toString(parametersConstraints) + ", result: " + resultConstraint + " (" + resultIndex + ")}";
   }
 
+  public List<ProgramState> statesAfterInvocation(List<SymbolicValue> invocationArguments, ProgramState programState, Supplier<SymbolicValue> svSupplier) {
+    List<ProgramState> results = new ArrayList<>();
+    results.add(programState);
+    for (int index = 0; index < invocationArguments.size(); index++) {
+      // FIXME : varargs method should be handled
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)